### PR TITLE
Video thumbnail

### PR DESCRIPTION
**Pull request by thecalcc**
NHUB-315

### CODE REVIEW

**Review by tomaskikutis — APPROVED** [review]
fix e2e before merging

**Comment by MarkLark86** [comment]
@thecalcc If this is for v2.6, please don't merge yet as `develop` is currently for v2.5

**Comment by MarkLark86** [comment]
There's more issues here. I will fix this in 2.4 in javascript and create a 2nd PR to do the same in typescript

**Pull request closed without merging**
